tests/db_engine_specs/crate_tests.py

```diff
@@ -26,7 +26,7 @@ def test_convert_dttm(self):
         DB Eng Specs (crate): Test conversion to date time
         """
         dttm = self.get_dttm()
-        assert CrateEngineSpec.convert_dttm("TIMESTAMP", dttm) == "1546398245678.9"
+        assert CrateEngineSpec.convert_dttm("TIMESTAMP", dttm) == "1546394645678.9"
```

Comment:
This needs to be changed back to UTC (I need to look into how to make this always render the timestamp as UTC despite local timezone).